client/lib/preferences-helper/boolean-preference.tsx

```diff
@@ -45,18 +47,18 @@ const BooleanPreference: FunctionComponent< Props > = ( { name, value } ) => {
 			{ value !== localValue && (
 				<>
 					<button
-						className="preferences-helper__save-pref-button"
+						className="preferences-helper__preference-save"
 						onClick={ savePreferenceChange }
 						disabled={ value === localValue }
```

Comment:
This `disabled` prop is always `false` and can be omitted, because the entire buttons fragment is rendered only when `value !== localValue`